torch/csrc/jit/passes/quantization.cpp

```diff
@@ -18,8 +18,30 @@ void PropagateQuantInfo(std::shared_ptr<Graph>& graph) {
   throw std::runtime_error("Pass not implemented yet!");
 }
 
+static void addObserverFor(torch::jit::Value* v) {
+  torch::jit::Node* def = v->node();
+  torch::jit::Node* observer =
+      def->owningGraph()->create(at::Symbol::fromQualString("aten::observer"));
+  def->owningBlock()->appendNode(observer);
+  observer->moveAfter(def);
+  v->replaceAllUsesWith(observer->outputs()[0]);
+  observer->outputs()[0]->setUniqueName(v->uniqueName() + ".observed");
+  observer->addInput(v);
```

Comment:
Comment: input of v is added here because adding it at node creation would make it a use of v and subject to replaceAllUsesWith